pkgs/servers/sql/postgresql/ext/sqlite_fdw.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  sqlite,
+  postgresql,
+}:
+
+stdenv.mkDerivation {
+  pname = "sqlite_fdw";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "pgspider";
+    repo = "sqlite_fdw";
+    rev = "v2.4.0";
+    hash = "sha256-u51rcKUH2nZyZbI2g3crzHt5jiacbTq4xmfP3JgqnnM=";
+  };
+
+  buildInputs = [
+    sqlite
+    postgresql
+  ];
+
+  makeFlags = [ "USE_PGXS=1" ];
+
+  installPhase = ''
+    mkdir -p $out/lib $out/share/postgresql/extension
+    mv *.so $out/lib
+    mv *.sql *.control $out/share/postgresql/extension
+  '';
+
+  meta = with lib; {
+    description = "SQLite Foreign Data Wrapper for PostgreSQL";
+    homepage = "https://github.com/pgspider/sqlite_fdw";
+    maintainers = with maintainers; [ apfelkuchen6 ];
+    platforms = platforms.unix;
```

Comment:
Please set platforms = linux or fix the Darwin build failure
```
       > Running phase: installPhase
       > mv: missing destination file operand after '/nix/store/kxzxm94b3di0acd7g1prcbfqp69s3112-sqlite_fdw-2.4.0/lib'
       > Try 'mv --help' for more information.
```

Reply:
Thanks, it builds on macOS with `*.so *.dylib`. I don't have a mac to test whether it actually works though.